pyccel/ast/datatypes.py

```diff
@@ -5,6 +5,10 @@
 # go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
 #------------------------------------------------------------------------------------------#
 
+"""
+Manupilating supprted datatypes in C/Fortran.
```

Comment:
Typo:
`Manupilating`->`Manipulating`
`supprted`->`supported`

But I am not sure that this is the right word? It seems to me that this module contains the definitions of classes handling datatypes but it doesn't seem to manipulate them